### DISCUSSION TITLE
How do I get a list of CVE issues between two releases?

### DISCUSSION THREAD

**Discussion by tomschr** [jquery/jquery#5079] [General]
Hi, :wave: 

For a package update, I need to update jQuery from 1.10.2 to 1.12.4. Unfortunately, I don't know much about jQuery or this repository.

So I was looking which CVE issues were fixed during this update to mention this in my package update. I was looking for a Changelog file and in the Git log, but I haven't found anything yet. I tried `git log 1.10.2..1.12.4`, but without success. I also searched in the GitHub discussion.

How can I get a list of all fixed CVE issues between these two releases? Where can I find them? Much appreciated for any pointers, many thanks!

**Comment by mgol** [comment]
You should be able to get that info from Snyk: https://security.snyk.io/package/npm/jquery
